README.md

```diff
@@ -88,28 +80,6 @@ You can use the [Helm CLI][installing-helm] to develop a chart and add it to thi
 
 > Feel free to add different values to the chart.
 
-### <a name='Automatedversionbumps'></a>Automated version bumps
-
-This repository is configured to accept webhook requests to bump chart versions. Upon receiving a version bump request, a GitHub Action generates a pull request with the requested changes. The pull request must still be merged manually.
-
-#### <a name='Triggeranautomatedversionbump'></a>Trigger an automated version bump
```

Comment:
This is still used by charts/synthetics-minion https://github.com/newrelic/helm-charts/pull/729

Unless we want to "deprecate" it by removing the docs

Reply:
I am adding it back